ckanext/datavicmain/actions.py

```diff
@@ -119,4 +120,17 @@ def datavic_user_create(context, data_dict):
         )
 
     log.debug('Created user {name}'.format(name=user.name))
-    return user_dict
\ No newline at end of file
+    return user_dict
+
+def nominate_resource_view(context, data_dict):
```

Comment:
It's better to add project-specific prefixes to custom actions. For example, `datavic_nominate_resource_view`